web/client-api/src/main/java/io/deephaven/web/client/api/tree/JsTreeTable.java

```diff
@@ -119,31 +118,44 @@ public class JsTreeTable extends HasLifecycle implements ServerObject {
      * all subsequent steps must be performed as well.
      */
     private enum RebuildStep {
-        FILTER, SORT, HIERARCHICAL_TABLE_VIEW, SUBSCRIPTION;
+        UPDATE_VIEW, FILTER, SORT, HIERARCHICAL_TABLE_VIEW, SUBSCRIPTION
+    }
+
+    public enum RollupNodeType {
+        UNSPECIFIED, AGGREGATED, CONSTITUENT
```

Comment:
Let's fold this into the string constants you created for CustomColumOptions - i'll make a patch to suggest what i mean.